src/scripts/import-projects.ts

```diff
@@ -148,13 +149,15 @@ export async function importProjects(
   }
 
   if (filteredTargets.length === 0) {
-    return { projects: [], targets, filteredTargets, skippedTargets: 0 };
+    return { projects: [], targets, filteredTargets, skippedTargets: 0, logFile: "" };
```

Comment:
return `null` or `undefined` instead like in the function `logImportedBatch`. Empty string is not a values that easy to check for.

Reply:
I prefer `undefined` aka not set at all